plivo/resources/brand.py

```diff
@@ -13,63 +13,69 @@ class Brand(PlivoResourceInterface):
-        email=[optional(of_type(six.text_type))],
-        entity_type=[optional(of_type(six.text_type))],
-        first_name=[optional(of_type(six.text_type))],
-        last_name=[optional(of_type(six.text_type))],
-        phone=[optional(of_type(six.text_type))],
-        postal_code=[optional(of_type(six.text_type))],
-        registration_status=[optional(of_type(six.text_type))],
-        state=[optional(of_type(six.text_type))],
-        stock_exchange=[optional(of_type(six.text_type))],
-        stock_symbol=[optional(of_type(six.text_type))],
-        street=[optional(of_type(six.text_type))],
-        vertical=[optional(of_type(six.text_type))],
-        website=[optional(of_type(six.text_type))],
-        secondary_vetting=[optional(of_type(six.text_type))])
+        brand_alias=[optional(of_type(six.text_type))],
+        brand_type=[optional(of_type(six.text_type))],
+        profile_uuid=[optional(of_type(six.text_type))],
+        secondary_vetting=[optional(of_type_exact(bool))],
+        url=[optional(of_type(six.text_type))],
+        method=[optional(of_type(six.text_type))],
+        subaccount_id=[optional(of_type(six.text_type))],
+        emailRecipients=[optional(of_type(six.text_type))],
+        campaignName=[optional(of_type(six.text_type))],
+        campaignUseCase=[optional(of_type(six.text_type))],
+        campaignSubUseCases=[optional(of_type_exact(list))],
+        campaignDescription=[optional(of_type(six.text_type))],
+        sampleMessage1=[optional(of_type(six.text_type))],
+        sampleMessage2=[optional(of_type(six.text_type))],
+        embeddedLink=[optional(of_type_exact(bool))],
+        embeddedPhone=[optional(of_type_exact(bool))],
+        numberPool=[optional(of_type_exact(bool))],
+        ageGated=[optional(of_type_exact(bool))],
+        directLending=[optional(of_type_exact(bool))],
+        subscriberOptin=[optional(of_type_exact(bool))],
+        subscriberOptout=[optional(of_type_exact(bool))],
+        subscriberHelp=[optional(of_type_exact(bool))],
+        affiliateMarketing=[optional(of_type_exact(bool))],
+        resellerID=[optional(of_type(six.text_type))])
     def create(self,
-               vertical,
-               street,
-               stock_symbol,
-               stock_exchange,
-               state,
-               registration_status,
-               postal_code,
-               phone,
-               entity_type,
-               email,
-               ein_issuing_country,
-               ein,
-               country,
-               company_name,
-               city,
-               alt_business_id_type=None,
-               alt_business_id=None,
-               first_name=None,
-               last_name=False,
-               website=None,
-               secondary_vetting=None):
+               brand_alias,
+               brand_type,
+               profile_uuid,
+               secondary_vetting,
+               url,
```

Comment:
i think, most of fields are option, can we set default value for the same. so that customer no need to specify all fields